app/assets/stylesheets/moderators.scss

```diff
@@ -213,36 +213,205 @@
     margin: 90px auto;
     border-radius: 12px;
   }
+
+  // Reaction Buttons START
+
+  .thumb-reactions {
+    display: flex;
+    justify-content: center;
+  }
+
   .reaction-button {
     position: relative;
-    width: 80px;
-    height: 80px;
     background: white;
     border-color: white;
-    margin: 30px 30px;
-    border-radius: 200px;
-    border: 0px;
-    padding: 12px 12px;
+    border: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 156px;
+    height: $su-10;
+    padding: $su-4 $su-6;
+    margin: $su-4 $su-2 $su-4 $su-4;
+    border-radius: $br-default;
+
+    &:last-child {
+      margin: $su-4 $su-4 $su-4 $su-2;
+    }
+
+    &:hover {
+      background: rgba(59, 73, 223, 0.05);
+    }
     &:active {
-      border: 0px;
+      border: 0;
+    }
+    &:hover .reaction-button-circle {
+      border-color: var(--accent-brand);
     }
   }
 
-  .reaction-button img {
+  .reaction-button-circle {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 0 auto;
+    width: $su-9;
+    height: $su-9;
+    border: solid 2px var(--base-20);
+    border-radius: 100%;
+    background: $white;
+  }
+
+  .reaction-button-circle img {
+    width: $su-7;
+  }
+
+  img.upvote {
+    -webkit-transform: rotate(180deg);
+    -moz-transform: rotate(180deg);
+    -ms-transform: rotate(180deg);
+    -o-transform: rotate(180deg);
+    transform: rotate(180deg);
+  }
+
+  .vote-text {
+    font-weight: $fw-bold;
+    font-size: $fs-base;
+    line-height: $lh-base;
+    margin-top: $su-2;
+    color: var(--base-90);
+  }
+
+  .reaction-checkmark {
+    opacity: 0;
     position: absolute;
-    top: 5px;
-    width: 70px;
-    left: 5px;
+    top: 63px;
```

Comment:
These values are very pixel-specific and would be hard to maintain, but I'm wondering if it would be possible for you to move the `.reaction-checkmark` so that it's a child element of `.reaction-button-circle`, and then set the absolute position of `.reaction-checkmark` relative to .reaction-button-circle. It might look like this:

```
.reaction-button-circle {
  position: relative;
  .position-checkmark {
    position: absolute;
    right: 0;
    bottom: 0;
  }
}
```

This way, we always know that the `.reaction-checkmark` is pinned to the bottom-right of the `.reaction-circle-button`